Assignment2/alvarogar4-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,42 @@
+{
+	"@context":{
+		"student":"http://estudiantesUPM.com/",
+		"fullName":"student:fullName",
+		"hometown":"student:hometown",
+		"age":"student:age",
+		"almaMater":"student:almaMater",
+		"parentsNames":"student:parentsNames",
+		"subjects":"student:subjects",
+			"name":"subjects:name",
+			"teachers":"subjects:teachers",
+			"description":"subjects:description"
```

Comment:
These subject attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes.
You don't need to change them, but remember to provide good definitions for the future.